packages/internals/src/__tests__/getGenerators/getGenerators.test.ts

```
@@ -201,15 +198,15 @@ describe('getGenerators', () => {
     `)
 
-    expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-    expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-    expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
-    expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
+    expect(consoleMock.log.mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
+    expect(consoleMock.info.mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
+    expect(consoleMock.warn.mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
+    expect(consoleMock.error.mock.calls.join('\n')).toMatchInlineSnapshot(`""`)
 
     generators.forEach((g) => g.stop())
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)